Files committed here are reports or presentations from NYU Master's projects. If you would like to see the code associated with each project, please reach out via email or direct message to request access. Thank you!

Details for each project are listed below:

## Biology and Biotechnology: Analyzing Glioblastoma Progression and Treatment Resistance
▪	Developed theoretical experimental studies to explore genetic, transcriptomic, and epigenetic variation in glioblastoma across timepoints to determine treatment resistance and potential therapeutic targets.

▪	Proposed to utilize data from whole genome sequencing (WGS), RNA-seq, and ChIP-Seq to examine variations and epigenetic modifications. 

## Biostatistics: Integrative Analysis Using PCA, Clustering, and Performance Metrics Across Diverse Datasets
▪	Collaboratively performed biostatistical analysis in R on NBA statistical data, and handwriting images, and gene expression data. 

▪	NBA data: standadized, sorted, visualized. Pearson correlation and heat map to visualize performance metric correlations.

▪	Handwriting data: normalized and PCA on pixel data. Variance calculated and plotted. Hierarchical clustering performed and visualized (dendogram and heatmap). Classification using Euclidean distance.

▪	Gene Expression data: unflitered distribution plot and histogram. Filtered based on expression thresholds and visualized distribtuion. Hierarchical clustering and heatmap of gene expression. DESeq2 analysis of DEGs.

▪	 Tools: scale, ggplot2, dplyr, pheatmap, prcomp, clust, hclust, sqrt, sum.

## Capstone: Binding Affinity Distribution in Protein-Ligand Pairs Between Substrates and Products
▪	Conducted protein-ligand binding affinity prediction and analysis using Python, PyMol, and AutoDock Vina.

▪	Performed mass spectrometry data analysis and database management and integration with Python and R, employing data visualization and statistical analysis.

▪	Tools: Bioservices, BS4, pandas, numpy, scikit-learn, Matplotlib; dplyr, ggplot2, stats, outliers

## Machine Learning: Predictive Modeling of Diabetes and Pre-Diabetes Risk Using Healthcare and Lifestyle Data
▪	Analyzed CDC diabetes dataset using Python, including data preprocessing for classifier implementation.  

▪	Compared Random Forest and Decision Tree Classifiers with and without PCA, and visualized performance metrics (confusion matrices, ROC curves, precision-recall curves).

## Next Generation Sequencing: Analysis of NRDE2 Depletion in MDA-MB-231 Cells Using RNA-Seq
▪	Utilized Unix bash scripting and workflow management systems to implement nf-core/rnaseq pipeline for Illumina sequence quality control, trimming, alignment and quantification.

▪	Utilized R programming language to perform DESeq2 analysis and visualization using PCA, MA, and other plots.

## Proteomics: Proteomic Analysis of CCL2 (MCP-1) in Retinal Inflammation
▪	Investigated the role of CCL2 (MCP-1) in inflammation and retinal health using databases and publications.

▪	Performed database integration (UniProt, STRING, NextProt, KEGG, Human Protein Atlas) to map CCL2 expression, interactions, and pathways. 

▪	Verified CCL2 expression in retina and identified Muller glia cells as primary source using RNA-Seq and FANTOM5 datasets.

▪	Assessed CCL2 and Rhodopsin interactions, discovering indirect associations through shared pathways and networks.

## Transcriptomics: scRNA-seq Analysis of Mouse Myeloid Cells in Tumor-Bearing Cerebella ± Radiation Treatment to Identify DEGs
▪	Utilized R and Seurat on RNA-seq data to run PCA, differential gene expression analysis, UMAP Clustering, and gene-ontology enrichment. 

▪	Data visualization as PCA plots, violin plots, tables, Venn diagrams, UpSet plots, feature plots, dimension plots, etc.
